designs/administrator/future-openshift/workloads-metrics-tab/index.md

```diff
@@ -0,0 +1,27 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Workloads Metrics tab
+
+
+# Background
+
+When moving to OCP 4 the metrics charts for Deployments, Deployment Configs, StatefulSets, DaemonSets, ReplicaSets, and ReplicationControllers were removed from the details pages. There has been feedback to reinstate these charts as they give users the ability to review activity at a glance and drill down to get additional information.
+
+
+## Goal
+
+The goal is to bring the utilization charts back into the OCP console while improving the functionality and user experience.
+
+
+## Designs for 4.x release
+
+The utilization charts will receive the following revisions:
+* The charts will be moved into their own **Metrics tab** __Strong__ and use a format more inline with the monitoring pages. Moving these charts off the details page allows more space for things like HPAs in the future.  
+* Updating the tooltip format to be consistent with Monitoring.
+* Using multi-line charts will show activity from multiple pods in a Deployment to help show comparisons and lower performing assets
+
```

Comment:
Maybe worth listing out what all charts will appear in this tab? It is the same set as the existing Details but thought it could be worth capturing here.